getsolus/packages
# yosys: Update to 0.36

**Summary**
There are numerous new features from 0.23 to 0.36, including improved support for Verific, verification, new commands and options.
- [0.36](https://github.com/YosysHQ/yosys/releases/tag/yosys-0.36)

**Test Plan**
  - passed all cases in the check step.
  - install, run example in examples/osu035 with make, see result file example.edif.
  - run example in examples/python-api, check if it produces a plot.

**Checklist**
